### DISCUSSION TITLE
백엔드 설계하면서 생긴 해결해야 할 질문들

### DISCUSSION THREAD

**Discussion by gomarag** [codestates/siphoon-note#46] [Q&A]
- [x] 개발, 테스트, 배포용 db를 어떻게 다르게 만들까?
  - 개발용 db에 더미데이터를 넣어두고 팀원들과 작업하면서 사용할 예정
  - 개발용 db에서 문제 없는 것을 확인 후 같은 구조로 배포용 db만들어서 배포용 서버와 연결할 예정
- [x] 액세스, 리프레시 모두 쿠키에 담아서 통신하기로 했는데, 리프레시 토큰을 유저 정보로 db가 저장한다면 영향을 받는 부분이 어디 생길까?
  - 리프레시 토큰은 db에, 액세스 토큰은 쿠키에
- [ ] 액세스 토큰 만료 시 유저에게 어떻게 안내해줘야 하나? -> 로그인 화면 보여주면서 다시 로그인 시키면 되나?
- [ ] OAuth로 가입한 유저의 회원정보는 서버가 어떻게 관리하나?
- [ ] 로그인 후 인증에 성공한다면 로그인 상태를 일정 기간 유지하기 위해서 어떤 방법을 써야 하나?

**Comment by Leekyeongmi** [comment]
오호.. 클라이언트 작업하면서 생기는 질문들도 이렇게 정리했다가 크루님들과 나누면 좋겠네요 :)
